node/src/chain_spec.rs

```diff
@@ -161,6 +162,7 @@ pub fn local_config() -> Result<ChainSpec, String> {
         let mut props = Properties::new();
         props.insert("tokenSymbol".into(), "ACME".into());
         props.insert("tokenDecimals".into(), 18.into());
+        props.insert("ss58Format".into(), zkv_runtime::SS58Prefix::get().into());
```

Comment:
Please insert `props.insert("ss58Format".into(), zkv_runtime::SS58Prefix::get().into());` also in `testnet_config_build` (line 232/233).
I know it's dead code, but if possible let's keep it aligned.